patches/third_party-blink-renderer-core-html-parser-BUILD.gn.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/third_party/blink/renderer/core/html/parser/BUILD.gn b/third_party/blink/renderer/core/html/parser/BUILD.gn
+index 16e082d52f6fcdfea1e36b49e2827b23293fec75..4785545bd8af5abf8dc729c2b82d1fee459d2eb8 100644
+--- a/third_party/blink/renderer/core/html/parser/BUILD.gn
++++ b/third_party/blink/renderer/core/html/parser/BUILD.gn
+@@ -4,6 +4,7 @@
+ 
+ import("//testing/libfuzzer/fuzzer_test.gni")
+ import("//third_party/blink/renderer/core/core.gni")
++import("//brave/vendor/speedreader_rust_ffi/config.gni")
```

Comment:
Curious if there's a way to do this outside of blink? Maybe @bridiver can take some time to look into it.